allennlp/tests/commands/fine_tune_test.py

```diff
@@ -47,7 +47,7 @@ def test_fine_tune_does_not_expand_vocab_by_default(self):
         model = load_archive(self.model_archive).model
 
         # By default, no vocab expansion.
-        fine_tune_model(model, params, self.serialization_dir)
+        train_model(params, self.serialization_dir, model=model)
```

Comment:
We shouldn't be calling `train_model` here directly in a `fine_tune_test.py` file.  If there are tests for functionality that got moved to `train.py`, those tests should be moved into `train_test.py`.  In general, the tests in a file should only call methods in the corresponding implementation (so, `fine_tune_model_from_file_paths`).